ui/tutorial-renderer/src/main/kotlin/org/cru/godtools/tutorial/layout/liveshare/TutorialLiveShareLayout.kt

```diff
@@ -79,16 +81,39 @@ internal fun TutorialLiveShareLayout(
 
     Spacer(modifier = Modifier.weight(1f))
 
-    Button(
-        onClick = {
-            when (page) {
-                Page.LIVE_SHARE_START -> onTutorialAction(Action.LIVE_SHARE_FINISH)
-                else -> nextPage()
+//
+
+    if (page == Page.LIVE_SHARE_START) {
```

Comment:
from a style perspective, I tend to like using when blocks better to switch between multiple options instead of an if-else.

Something along the lines of:
```kotlin
when (page) {
    Page.LIVE_SHARE_START -> Row {
        ...
    }

    else -> Button {
        ...
    }
}
```